src/app/routes/getInitialData/index.js

```diff
@@ -30,6 +31,7 @@ const getInitialData = async ({ match }) => {
 
     if (status === 200) {
       data = await response.json();
+      data = preprocess(data);
```

Comment:
Could we maybe try catch this? If preprocess fails, use the ares response clean

Reply:
Done in the `preprocess` function itself 👍